glossary/product-definition.md

```diff
@@ -4,4 +4,6 @@ type: Resource
 
 # Product
 
-A re-implementable set of public code, packaged with documentation. We support products over projects to acknowledge that code is not something you make and then abandon - it is not just about planning and control, it is about innovation and maintenance.
+A product is an object or system made available for consumer use that can be offered to a market to satisfy the desire or need of a customer. A product can be classified as tangible or intangible. A tangible product is a physical object that can be perceived by touch such as a building, vehicle, gadget, or clothing. An intangible product is a product that can only be perceived indirectly such as an insurance policy. Services can be broadly classified under intangible products which can be durable or non durable.
```

Comment:
Is this paragraph needed? It covers others definition of what product is, but in the paragraph below we give our definition of it. I am afraid this might confuse people on what this glossary actually means. If a word link here from somewhere else on About, do we aim to explain that with the first or second paragraph?

Reply:
removed generic product definition